docs/content/en/docs/operator-manual/control-plane/metrics.md

```diff
@@ -18,15 +18,35 @@ To enable monitoring system for PipeCD, you first need to set the following valu
 --set monitoring.enabled=true
 ```
 
-
-### Grafana dashboard
+## Dashboards
 If you've already enabled monitoring system in the previous section, you can access Grafana using port forwarding:
 
 ```
 kubectl port-forward -n {NAMESPACE} svc/{PIPECD_RELEASE_NAME}-grafana 3000:80
 ```
 
-### Alert notifications
+#### Control-plane dashboards
```

Comment:
It would be nice if we could have some screenshots.

Reply:
We're on the same page. I'm looking to add dashboard screenshots after finishing up it (there are a couple of things to improve actually)